### DISCUSSION TITLE
Pfad von HistoryGraph-Bilder

### DISCUSSION THREAD

**Discussion by exaware** [Zefau/ioBroker.jarvis#2015] [Q&A]
Hallo,
ich möchte meine HistoryGraphen per telegram verschicken. Hierzu benötige ich den Pfad, in dem die Bilder der HistoryGraphen liegen. Kennt jemand den Pfad?
Oder liegen die Graphen überhaupt als Bilder vor? Da man ja zoomen und Zeiträume ändern kann. In Chrome kann man die Graphen als Bild abspeichern, bin mir aber nicht sicher ob Chrome den Graph in ein Bild umwandelt.

**Comment by Zefau** [comment]
Die liegen leider nicht als Bild vor.
